### PR TITLE
feat(chat-e2e): overlay sandbox component

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

overlay sandbox component

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/546266)
>Environment URL: https://chat-ai-dial-chat-pr-2776.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2776.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2776.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2776.nightly-test.deltixhub.io
>E2E tests status: success

**Review by nartovm — DISMISSED** [review]

File: apps/overlay-sandbox/app/page.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Link from 'next/link';
 export default async function Index() {
   return (
     <div>
-      <Link href={'/cases/overlay'}>Overlay</Link>
+      <Link href={'/cases/overlay/modelIdSetSandbox'}>Overlay</Link>
```
Comment (IlyaBondar) [inline apps/overlay-sandbox/app/page.tsx]: Why do you move it in `/modelIdSetSandbox`? At least it should be in kebab-case as `overlay-manager`, but I think previous `/cases/overlay` was better

Comment (irinakartun) [reply apps/overlay-sandbox/app/page.tsx]: for the testing purposes sandboxes with different options and combination of feature flags should be launched. I gonna put all the sandboxes inside `/cases/overlay/` that's why `modelIdSetSandbox`, which is used for verification of `modelId` option, was placed there. 
Fixed incorrect case

**Review by IlyaBondar — APPROVED** [review]

**Review by nartovm — APPROVED** [review]

**Pull request merged**